src/services/statemanager.js

```diff
@@ -110,13 +129,24 @@ ngeo.StateManager.prototype.getInitialValue = function(key) {
  * @param {Object.<string, string>} object Object.
  */
 ngeo.StateManager.prototype.updateState = function(object) {
-  this.ngeoLocation.updateParams(object);
+  var locationObject = {};
+  Object.keys(/** @type {!Object} */(object)).forEach(copyWithoutExcludedKeys,
```

Comment:
Instead of casing the object to `{!Object}` the code could be changed with:
```diff
diff --git a/src/services/statemanager.js b/src/services/statemanager.js
index a77c733..95043d4 100644
--- a/src/services/statemanager.js
+++ b/src/services/statemanager.js
@@ -17,7 +17,7 @@ ngeo.StateManager = function(ngeoLocation) {
 
   /**
    * Object representing the application's initial state.
-   * @type {Object.<string ,string>}
+   * @type {!Object.<string ,string>}
    */
   this.initialState = {};
 
@@ -107,7 +107,7 @@ ngeo.StateManager.prototype.getInitialValue = function(key) {
 
 /**
  * Update the application state with the values in `object`.
- * @param {Object.<string, string>} object Object.
+ * @param {!Object.<string, string>} object Object.
  */
 ngeo.StateManager.prototype.updateState = function(object) {
   this.ngeoLocation.updateParams(object);
```